aether-ui/aether/ui/assets/apps/pipeline/components/DeleteModal.jsx

```diff
@@ -19,9 +19,28 @@
  */
 
 import React, { Component } from 'react'
-import { FormattedMessage } from 'react-intl'
+import { FormattedMessage, FormattedHTMLMessage, injectIntl, defineMessages } from 'react-intl'
 import { Modal } from '../../components'
 
+const MESSAGES = defineMessages({
+  delete: {
+    id: 'modal.delete.text',
+    defaultMessage: 'Delete {obj} <b>{objName}</b>'
+  },
+  submissions: {
+    id: 'modal.delete.submissions.text',
+    defaultMessage: 'Data <b>submitted to</b> this pipeline (Submissions)'
+  },
+  entities: {
+    id: 'modal.delete.entities.text',
+    defaultMessage: 'Data <b>created by</b> this {obj} (Entities)'
+  },
+  objectType: {
+    id: 'modal.delete.object.type',
+    defaultMessage: '{objType}'
```

Comment:
:disappointed:

Reply:
```javascript
const MESSAGES = defineMessages({
  // ...
  pipeline: {
    id: 'modal.delete.object.type.pipeline',
    defaultMessage: 'pipeline'
  },
  contract: {
    id: 'modal.delete.object.type.contract',
    defaultMessage: 'contract'
  }
})
```

Reply:
I rather have it as a template with interchangeable variables than having a constants, except if there's a side effect am not seeing?

Reply:
The extracted text would be "{my_variable}" but never "contract" or "pipeline", the purpose of all of this is to make those texts "eligible" for translation... otherwise this would be much simpler. :books: